### PR TITLE
Entities landing page experiment

### PR DESCRIPTION
**Pull request by lmilan**

### CODE REVIEW

File: app/_kong_entities/service.md
Comment (lena-larionova) [inline app/_kong_entities/service.md:8]: I like this a lot. We can then require page to have a description (maybe limit it to some character count)? Then we can pull it in any place that we do link previews, including SEO reasons - this is the description that should get pushed to Google.

File: app/_includes/landing_pages/entity_card.md
Comment (fabianrbz) [inline app/_includes/landing_pages/entity_card.md:4]: this is great!
you can actually call collections by their name, e.g. `site.kong_entities`.

Comment (fabianrbz) [reply app/_includes/landing_pages/entity_card.md:4]: weird, we are using it [here](https://github.com/Kong/developer.konghq.com/blob/main/app/kong-entities.html#L7).
which part isn't working for you? the `where_exp` or fetching the collection?

Comment (lmilan) [reply app/_includes/landing_pages/entity_card.md]: I tried again and it works now, thanks! I think originally I may have only tried `site.collections.kong_entities` 😅

**Review by lena-larionova — APPROVED** [review]
lgtm, in case you're waiting on approval 😄

**Pull request merged**
